Conf attending cheatsheet and updates to style and paths for pngs (#18)
jhudsl/cheatsheets: attending_conferences.Rmd

<div class="row">

<div class="column">
<center><img src="resources/images/unhappy.png" width = 50% > </center>
<br>
Conferences can feel intimidating, especially your first few.
</div>
<div class="column">
<center><img src="resources/images/happy.png" width = 50%> </center> 
<br>
But don't worry, this is a common feeling.

> You deserve to be attend and no one knows what all the talks are about.

Reviewer: ```suggestion
You deserve to be attending and no one completely understands every talk.
```

Author: updated this

<div class="column">

# What to expect:

- Conferences cover a ton of information! (It may feel overwhelming)
- Talks conclude with time for questions
- Plenary talks are large talks typically for the full conference with experienced speakers
- People will attend talks and some will take notes
- People will come and go
- People will generally ask constructive questions
- Occasionally people might ask aggressive questions, but this is rare

</div>
<div class = "column">
<center>
::: Note

# Possible Goals:

<input type="checkbox"> Meet new people 

<input type="checkbox"> Share your research

<input type="checkbox"> Look for a job

<input type="checkbox"> Get ideas for new projects

<input type="checkbox"> Find collaborators
:::
</center>
</div>
</div>

<hr>

<div class="row">

<div class="column">
# Best Practices:

> <input type="checkbox">  Don't try to absorb everything! <img src="resources/images/face_explode.png" width = 8%>

Reviewer: The image here isn't loading in the preview

Author: this should be fixed

- Focus on your interests, but also check out new topics.

<input type="checkbox">  Take notes <img src="https://www.iconpacks.net/icons/free-icons-6/free-icon-notes-and-pencil-22337.png" width = 6%>

- It can be difficult to retain what you learn, so take notes. Include information about the researchers and relevant papers, so you can reach out or learn more.

<input type="checkbox">  Get ideas <img src="https://www.iconpacks.net/icons/4/free-icon-light-bulb-13629.png" width = 8%>

- Conferences are a great places to make new connections.
</div>

<div class="column">

<br>
<br>
<br>
<input type="checkbox">  Talk to people <img src="https://www.iconpacks.net/icons/4/free-icon-conversation-11886.png" width = 8%>

-  You never know who you might be sitting next to! Find new collaborators, people who know about helpful new methods, opportunities, jobs, etc.
-  Start with small talk, talk about a presentation, or talk about your work
- Transition times, after talks, and during breaks are good opportunities to meet people

> - Talk to speakers, ask questions (this can be after a talk or during the talk)

Reviewer: What do you mean by after the talk? Like during a Q&A time or after the session, just sometime during the conference?

Author: I meant after sessions but simplified to transition times and during breaks
